test/SynTreeSpec.hs

```diff
@@ -78,6 +81,11 @@ invalidBoundsSynTree = do
 
 spec :: Spec
 spec = do
+  describe "config" $
+    it "default config should pass config check" $
+      ioProperty $ do
+        output <- run English (checkSynTreeConfig defaultSynTreeConfig)
+        return $ isJust output
```

Comment:
Looks like using `Control.Monad.Output.Debug.checkConfigWith` would be more idiomatic.

Reply:
Or using the fact that there is also a pure (non-`IO`) version of a "runnable output monad" (https://github.com/fmidue/output-monad/blob/d469ed0b22df718dde4d4d718a7f6fd54cfe2a0d/output-monad/src/Control/Monad/Output/Generic.hs#L253-L256) which together with `evalLangM` from there should allow to write this test without needing `ioProperty`.

Reply:
I dont quite get how to use these in order to get a `Bool` at the end.

Reply:
I'm just going by the types. Running code rather rarely. :smile:

In this case, the types seem to entail that if `x :: forall m. OutputMonad m => LangM m`, then `runIdentity (evalLangM x) :: Maybe ()` can be used. So, an `isJust` on this should correspond to no `reject` having happened.